src/client/containers/layout.js

```diff
@@ -123,12 +123,33 @@ class Layout extends React.Component {
 			</span>
 		);
 
+		const privilegesDropdownTitle = (
+			<span>
+				<FontAwesomeIcon icon={faShieldHalved}/>
+				{' Privileges'}
```

Comment:
This doesn't need to be quoted and in a JSX block, it can just be the text content